static/js/components/CommentVoteForm.js

```diff
@@ -55,9 +80,38 @@ export default function CommentVoteForm(props: Props) {
   const downvoted = comment.downvoted !== downvoting && !upvoting
   const upvoted = comment.upvoted !== upvoting && !downvoting
 
+  // here we create an optimistic score based on assuming that the action in
+  // progress (if any) will succeed
+  let { score } = comment
```

Comment:
Can you add a check for whether the comment author is the user? Upvoting/downvoting your own comment does not affect the score. When the user votes on their own comment this causes an incorrect score to flash before reverting to the previous score

Reply:
yeah, that's a good idea, I'll add that in